src/haddock/modules/analysis/caprieval/defaults.yml

```diff
@@ -0,0 +1,101 @@
+basic:
```

Comment:
doesn't it make more sense if each parameter have a `level` instead of categorizing by it?

Reply:
Is it now easier to see all `basic` in a row? If you open the config file and want to see "which are all `intermediate` parameters", how do you do if params have a `level` key instead?

From the python parsing perspective is the same, there's no problem changing it.

Reply:
just a matter of preference I guess, for me would be better to have all parameters listed and each one would have a level definition, maybe less error-prone?

When someone is adding a new parameter i have a feeling they'll just add it in the same level as you define the levels now.

Reply:
It is possible also. We can always add `# basic` kind of comment to separate them visually. I am agnostic, to be honest. Either way is fine for me. I will wait for @amjjbonvin final verdict before changing the implementations. Alexandre will be the one changing these files the most.

Reply:
I see three ways
1. parameters nested within level
```yaml
basic:
 parameter1: ...
```

Each parameter belongs to 1 level and each greater level should include parameters of the smaller level.

2. levels per parameter
````
parameter1:
  ...
  levels: 
    - basic
````

It is harder to get overview which parameters are required for a level, but CLI, generated docs can help alleviate this.

3. levels outside parameters
```
parameters:
  parameter1: ...
levels:
  basic:
  - parameter1
```

The parameter keys need to be repeated

> A side note YAML can use [anchors and aliases](https://yaml.org/spec/1.2.2/#71-alias-nodes) to reduce duplication.

Reply:
We have solved thid using the `explevel` key for each parameter.

Reply:
We have solved this adding an `explevel` to each parameter.